bundles/net.tourbook/src/net/tourbook/tour/TourInfoUI.java

```diff
@@ -15,6 +15,9 @@
  *******************************************************************************/
 package net.tourbook.tour;
 
+import static org.eclipse.swt.events.ControlListener.controlResizedAdapter;
+import static org.eclipse.swt.events.SelectionListener.widgetSelectedAdapter;
```

Comment:
This was intentionally **NOT** used because when there is an error in this file, these imports are removed by Eclipse and all related imports are broken -> even more errors in this file

Reply:
I've never had issues with those but I have reverted them.

Reply:
I cannot remember if this happened since the beginning but since I found when this issue happens, I don't use the static imports for these listerners


https://github.com/mytourbook/mytourbook/assets/1283445/914f1aa7-a250-487b-bce3-7ca18f837bcf

Reply:
I think it's an Eclipse configuration issue on your Eclipse because I've never had this problem

Uploading Untitled Project.mp4…

Reply:
I already searched in the Eclipse save actions but have not found the reason why this happens